src/Sunburst.ts

```diff
@@ -26,6 +26,9 @@
 
 "use strict";
 
+import "core-js/stable";
```

Comment:
remove core-js and regenerator.
PowerBI doesn't support outdated browsers anymore

Reply:
Done: [7b82e3f](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/7b82e3f4570ec5d0b0f7589707aa4f1dbb900d90)